static/templates/message_edit_form.handlebars

```diff
@@ -5,8 +5,8 @@
     <div class="control-group">
         <label class="control-label edit-control-label" for="message_edit_topic">{{t "Topic" }}</label>
         <div class="controls edit-controls">
-          <input type="text" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
-          <select class='message_edit_topic_propagate' style='display:none;'>
```

Comment:
ah, the reason the display:none is there is because we don't want this dropdown to appear unless they are editing the topic (the display:none is changed by javascript once the user starts typing in the topic box)